include/zephyr/posix/signal.h

```diff
@@ -12,6 +12,61 @@
 extern "C" {
 #endif
 
+#ifdef CONFIG_POSIX_SIGNAL
+#define _NSIG       64
```

Comment:
Instead of _NSIG hard-coded can we use a Kconfig? Be sure to build assert that it is greater than or equal to sig rt min